recipes/mariadb-connector-c/all/test_package/test_package.c

```diff
@@ -1,11 +1,11 @@
 #include <mysql.h>
 #include <stdio.h>
 
-// See #4530
-#include <mysqld_error.h>
```

Comment:
Why is this removed? it's a test for bug some consumers faced 🤔

Reply:
Restore